.gitattributes

```diff
@@ -1,2 +1,3 @@
 # Auto detect text files and perform LF normalization
 * text=auto
+.* export-ignore
```

Comment:
I haven't investigated the syntax, is it possible to explicitly add an otherwise ignored file, in case we do have some file starting . that is to be exported? Otherwise maybe explicitly add files to be ignored.